pkg/protocol/engine/utxoledger/snapshot_test.go

```diff
@@ -197,17 +196,18 @@ func TestManager_Import(t *testing.T) {
 
 	output1 := tpkg.RandLedgerStateOutput()
 
-	require.NoError(t, manager.AddUnspentOutput(output1))
-	require.NoError(t, manager.AddUnspentOutput(tpkg.RandLedgerStateOutput()))
-	require.NoError(t, manager.AddUnspentOutput(tpkg.RandLedgerStateOutput()))
-	require.NoError(t, manager.AddUnspentOutput(tpkg.RandLedgerStateOutput()))
-	require.NoError(t, manager.AddUnspentOutput(tpkg.RandLedgerStateOutput()))
+	require.NoError(t, manager.AddGenesisUnspentOutput(output1))
+	require.NoError(t, manager.AddGenesisUnspentOutput(tpkg.RandLedgerStateOutput()))
+	require.NoError(t, manager.AddGenesisUnspentOutput(tpkg.RandLedgerStateOutput()))
+	require.NoError(t, manager.AddGenesisUnspentOutput(tpkg.RandLedgerStateOutput()))
+	require.NoError(t, manager.AddGenesisUnspentOutput(tpkg.RandLedgerStateOutput()))
 
 	ledgerIndex, err := manager.ReadLedgerIndex()
 	require.NoError(t, err)
 	require.Equal(t, iotago.SlotIndex(0), ledgerIndex)
 
 	mapDBAtIndex0 := mapdb.NewMapDB()
+	// Copy the current manager state to the mapDBAtIndex0
 	require.NoError(t, kvstore.Copy(mapDB, mapDBAtIndex0))
```

Comment:
There is also a CopyBatched method btw, in case we want to speed things up in some cases. Don't know how big the source is here.